python/ray/tests/BUILD

```diff
@@ -310,7 +310,7 @@ py_test_module_list(
 )
 
 py_test_module_list(
-    size = "medium",
+    size = "large",
```

Comment:
If only only test_basic.py times out, should we move it down together with test_basic_3?

Reply:
good idea

Reply:
updated